pom.xml

```diff
@@ -22,56 +22,84 @@
     ~ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.osiam</groupId>
-        <artifactId>osiam-server-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
-    </parent>
-
+    <groupId>org.osiam</groupId>
     <artifactId>osiam-resource-server</artifactId>
```

Comment:
I don't think that's an impact. if OSIAM is installed with the distribution or by downloading the war files, nobody will notice this. If the arctifactId will be renamed, maybe someone will notice that the server is split into 2 repos :)